index.html

```diff
@@ -9,7 +9,7 @@
     <style> @import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap'); </style>
 </head>
 <body>
-    <header>
+    <header class="header">
```

Comment:
- Please place the `section` tag, which has the class name `banner`, and all its content,  outside the header tag. Please fix this as it would help improve the semantics of your page 😄